interface/js_binding/ktx_wrapper.cpp

```diff
@@ -22,6 +49,136 @@ namespace ktx
         texture(texture&) = delete;
         texture(texture&& other) = default;
 
+        static texture createFromBuffer(const emscripten::val& data, int width, int height, int comps, bool srgb) {
```

Comment:
Bracket this function with `#if KTX_FEATURE_WRITE`.

Reply:
Isn't this an upload path? Why hide it behind `KTX_FEATURE_WRITE`?

Reply:
It is for creating a new texture rather than loading one from a KTX file so I'd always considered this part of the write functionality but now I see that the native equivalent is defined in `texture2.c` not `writer2.c`. For equivalence, we should not bracket this function.